tests/distributed/test_comm_ops.py

```diff
@@ -2,10 +2,9 @@
 
 Run `pytest tests/distributed/test_comm_ops.py --forked`.
 """
-from multiprocessing import Process, set_start_method
-
 import pytest
 import torch
+import ray
```

Comment:
Just curious, why change to ray?

Reply:
in the code below, i added a comment saying ray gives way better log for debugging purpose (i couldn't figure the failures from multiprocessing)

Reply:
gotcha! makes a lot of sense.